prime-router/src/test/kotlin/fhirengine/engine/FhirRouterTests.kt

```diff
@@ -230,20 +235,33 @@ class FhirRouterTests {
     @Test
     fun `test adding receivers to bundle`() {
         // set up
-        val bundle = FhirTranscoder.decode(valid_fhir_with_provenance)
+        val bundle = FhirTranscoder.decode(validFhirWithProvenance)
         val receiversIn = listOf(oneOrganization.receivers[0])
 
         // act
         FHIRBundleHelpers.addReceivers(bundle, receiversIn)
 
-//        val stringDoc = FhirTranscoder.encode(bundle)
-//        val reparse = FhirTranscoder.decode(stringDoc)
-
         // assert
         val provenance = bundle.entry.first { it.resource.resourceType.name == "Provenance" }.resource as Provenance
         val outs = provenance.target
         val receiversOut = outs.map { (it.resource as Endpoint).identifier[0].value }
         assert(receiversOut.isNotEmpty())
         assert(receiversOut[0] == "co-phd.full-elr-hl7")
     }
+
+    @Test
+    fun `test skipping inactive receivers`() {
+        // set up
+        val bundle = FhirTranscoder.decode(validFhirWithProvenance)
+        val receiversIn = listOf(oneOrganization.receivers[1])
+
+        // act
+        FHIRBundleHelpers.addReceivers(bundle, receiversIn)
+
+        // assert
+        val provenance = bundle.entry.first { it.resource.resourceType.name == "Provenance" }.resource as Provenance
+        val outs = provenance.target
+        val receiversOut = outs.map { (it.resource as Endpoint).identifier[0].value }
+        assert(receiversOut.isEmpty())
```

Comment:
Shouldn't we have here one receiver since one is testing.

Reply:
not in that case since the test only uses the inactive receiver. I will add another test case to check a mixed scenario